components/shared/Editor.tsx

```diff
@@ -315,8 +319,6 @@ export default function MockupEditor() {
 
       drawBackgroundImage(ctx);
     }
-
-    requestAnimationFrame(drawCanvas);
```

Comment:
Hey @kvtemadden, is requestAnimation not required anymore>

Reply:
It seems not, I believe handling the image loading more efficiently resolved the situation without needing to use `requestAnimationFrame`. I had originally tried that route on the initial fix as many people suggested this as the resolution when I was researching, but hadn't got it quite right. It seems fresh eyes resolved the problem & saved using it unnecessarily.

Reply:
Okay, this is fantastic! I can definitely see a reduction in RAM usage on the preview. It looks like the adjustments you made to handle image loading more efficiently have really helped.

Great work!